sampleDriver/Android.bp

```diff
@@ -61,11 +61,7 @@ cc_binary {
         "libui",
         "libutils",
         "libcutils",
-        "libyuv",
-        "libuws",
```

Comment:
Hi @DKlimenko, you've added some of the commits in a separate commit of this same PR - https://github.com/xen-troops/android_vendor_xen_evs/pull/9/commits/8f8a0105e307e39185aac7f7036a2ad4d2829716#diff-d84cecf0528f7adf07b53b01cb25d3a78aeca1e7ab0606d73204252687bf2363R65

Please, remove the "experiments" and leave only atomic valuable changes, without "fixing the bugs".